internal/store/pod.go

```diff
@@ -1511,6 +1511,27 @@ var (
 					}
 				}
 
+				return &metric.Family{
+					Metrics: ms,
+				}
+			}),
+		),
+		*generator.NewFamilyGenerator(
+			"kube_pod_runtimeclass_handler",
```

Comment:
Can we add a unit test for this, thanks! :)

Reply:
As per best practices the name should be `kube_pod_runtimeclass_handler_info`, can you adjust that as well, thanks!